templates/indices/timestampMs.go

```diff
@@ -0,0 +1,79 @@
+package indices
+
+// TimestampMs will hold the configuration for the timestampMs field
+var TimestampMs = Object{
+	"properties": Object{
+		"timestampMs": Object{
+			"type":   "date",
+			"format": "epoch_millis",
+		},
+	},
+}
+
+var TokensTimestampMs = Object{
```

Comment:
add comments to the other fields in this file

Reply:
added